### PR TITLE
 OCPBUGS-24030: add uncordon to mcn

### PR DESCRIPTION
**Pull request by cdoern**
we were using cordon to describe both cordon and uncordon in the MCN status

### CODE REVIEW

**Comment by rioliu-rh** [comment]
@cdoern when I try to list additional fields I cannot see Uncordoned, but I can find it in conditions
```console
$ oc get machineconfignode ip-10-0-117-1.us-east-2.compute.internal -o yaml | yq '.status.conditions[].type'
"Resumed"
"UpdatePrepared"
"UpdateExecuted"
"UpdatePostActionComplete"
"UpdateComplete"
"UpdateCompatible"
"Drained"
"AppliedFilesAndOS"
"Cordoned"
"RebootedNode"
"ReloadedCRIO"
"Updated"
"Uncordoned"

$ oc get machineconfignode ip-10-0-117-1.us-east-2.compute.internal -o wide
NAME                                       UPDATED   UPDATEPREPARED   UPDATEEXECUTED   UPDATEPOSTACTIONCOMPLETE   UPDATECOMPLETE   RESUMED   UPDATECOMPATIBLE   UPDATEDFILESANDOS   CORDONEDNODE   DRAINEDNODE   REBOOTEDNODE   RELOADEDCRIO
ip-10-0-117-1.us-east-2.compute.internal   True      False            False            False                      False            False     False              False               False          False         False          False
```

**Comment by rioliu-rh** [comment]
verify with latest code
now I can see column `UNCORDONEDNODE` in output of `oc get machineconfignode -o wide`
the condition can be updated
```console
$ mcn ip-10-0-15-42.us-east-2.compute.internal -o wide -w
NAME                                       UPDATED   UPDATEPREPARED   UPDATEEXECUTED   UPDATEPOSTACTIONCOMPLETE   UPDATECOMPLETE   RESUMED   UPDATECOMPATIBLE   UPDATEDFILESANDOS   CORDONEDNODE   DRAINEDNODE   REBOOTEDNODE   RELOADEDCRIO   UNCORDONEDNODE
ip-10-0-15-42.us-east-2.compute.internal   False     True             Unknown          False                      False            False     True               False               True           Unknown       False          False          False
ip-10-0-15-42.us-east-2.compute.internal   False     True             Unknown          False                      False            False     True               False               True           True          False          False          False
ip-10-0-15-42.us-east-2.compute.internal   False     True             Unknown          False                      False            False     True               False               True           True          False          False          False
ip-10-0-15-42.us-east-2.compute.internal   False     True             Unknown          False                      False            False     True               False               True           Unknown       False          False          False
ip-10-0-15-42.us-east-2.compute.internal   False     True             Unknown          False                      False            False     True               False               True           True          False          False          False
ip-10-0-15-42.us-east-2.compute.internal   False     True             Unknown          False                      False            False     True               Unknown             True           True          False          False          False
ip-10-0-15-42.us-east-2.compute.internal   False     True             True             False                      False            False     True               True                True           True          False          False          False
ip-10-0-15-42.us-east-2.compute.internal   False     True             True             Unknown                    False            False     True               True                True           True          Unknown        False          False
ip-10-0-15-42.us-east-2.compute.internal   False     True             True             True                       False            False     True               True                True           True          True           False          False
ip-10-0-15-42.us-east-2.compute.internal   False     True             True             True                       False            True      True               True                True           True          True           False          False
ip-10-0-15-42.us-east-2.compute.internal   False     True             True             True                       True             True      True               True                True           True          True           False          True
ip-10-0-15-42.us-east-2.compute.internal   False     True             True             True                       True             True      True               True                True           True          True           False          True
ip-10-0-15-42.us-east-2.compute.internal   True      False            False            False                      False            False     False              False               False          False         False          False          False
```

File: templates/master/01-master-kubelet/_base/files/kubelet.yaml [hunk 1]
```diff
@@ -31,6 +31,7 @@ contents:
       AzureWorkloadIdentity: true
       BuildCSIVolumes: true
       CloudDualStackNodeIPs: true
+      DisableKubeletCloudCredentialProviders: false
```
[the remark this answers is not in the document]
Comment (cdoern) [reply templates/master/01-master-kubelet/_base/files/kubelet.yaml:34]: this is to bring in some of the new featuregates that have been added to the default list

[the remark this answers is not in the document]
Comment (djoshy) [reply templates/master/01-master-kubelet/_base/files/kubelet.yaml:34]: ack, sorry didn't catch that!

**Pull request merged**